src/pages/Dashboard/EarnDashboard.tsx

```diff
@@ -0,0 +1,22 @@
+import { Main } from 'components/Layout/Main'
+import { DeFiEarn } from 'components/StakingVaults/DeFiEarn'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+
+import { DashboardHeader } from './components/DashboardHeader'
+
+export const EarnDashboard = () => {
+  return (
+    <Main headerComponent={<DashboardHeader />}>
+      <DeFiEarn
+        positionTableProps={{
+          filterBy: opportunities =>
+            opportunities.filter(opportunity => bnOrZero(opportunity.fiatAmount).gt(0)),
+        }}
+        providerTableProps={{
+          filterBy: opportunities =>
+            opportunities.filter(opportunity => bnOrZero(opportunity.fiatAmount).gt(0)),
+        }}
```

Comment:
We should use the right selector to diff. between the earn dashboard and rewards dashboard, not filtering like so